adr_web_app/app.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask,render_template,Markup

```

Comment:
Please change the directory name that contains all this to just `app`.

Reply:
Ok :)

Reply:
hmm, I am in trouble :(. This is what I did:
- pull rebase  in the master
- checkout to this pull branch
- made changes ,add and commit
-rebase from upstream
I got a conflict in setup.py, I resolved it. 
now this is what I am in
![](https://i.imgur.com/NmBWmAv.png)

What do you suggest me to do?